spec/helpers/vita_partner_helper_spec.rb

```diff
@@ -115,26 +115,26 @@
     end
 
     context "when the user's role is Greeter" do
-      let!(:site) { create :site, parent_organization: first_org }
       let!(:greeter) { create :greeter_user }
-      let(:first_org) { create :organization }
-      before do
-        greeter.role.update(organizations: [first_org, (create :organization)] )
-      end
+      let(:first_org) { create :organization, allows_greeters: true }
+      let!(:site) { create :site, parent_organization: first_org }
+      let(:second_org) { create :organization, allows_greeters: true }
```

Comment:
Consider adding a `allows_greeters: false` organization too, to test the full behavior of `.accessible_vita_partners` in this case.